catalog/nsili/catalog-nsili-transformer/src/main/java/org/codice/alliance/nsili/transformer/DAGConverter.java

```diff
@@ -349,38 +291,38 @@ private void addNsilCardAttribute(MetacardImpl metacard, Node node) {
     private void addNsilCommonAttribute(MetacardImpl metacard, Node node) {
         switch (node.attribute_name) {
         case NsiliConstants.DESCRIPTION_ABSTRACT:
-            metacard.setDescription(getString(node.value));
+            addDescription(metacard, getString(node.value));
             break;
         case NsiliConstants.IDENTIFIER_MISSION:
-            metacard.setAttribute(new AttributeImpl(NsiliMetacardType.IDENTIFIER_MISSION,
+            metacard.setAttribute(new AttributeImpl(IsrAttributes.MISSION_ID,
                     getString(node.value)));
             break;
         case NsiliConstants.IDENTIFIER_UUID:
             metacard.setId(getString(node.value));
             break;
         case NsiliConstants.IDENTIFIER_JC3IEDM:
-            metacard.setAttribute(new AttributeImpl(NsiliMetacardType.ID_JC3IEDM,
-                    getLong(node.value)));
+            metacard.setAttribute(new AttributeImpl(IsrAttributes.JC3IEDM_ID,
+                    getInteger(node.value)));
             break;
         case NsiliConstants.LANGUAGE:
-            metacard.setAttribute(new AttributeImpl(NsiliMetacardType.LANGUAGE,
+            metacard.setAttribute(new AttributeImpl(CoreAttributes.LANGUAGE,
                     getString(node.value)));
             break;
         case NsiliConstants.SOURCE:
-            metacard.setAttribute(new AttributeImpl(NsiliMetacardType.SOURCE,
+            metacard.setAttribute(new AttributeImpl(IsrAttributes.PLATFORM_NAME,
                     getString(node.value)));
             break;
         case NsiliConstants.SUBJECT_CATEGORY_TARGET:
-            metacard.setAttribute(new AttributeImpl(NsiliMetacardType.SUBJECT_CATEGORY_TARGET,
+            metacard.setAttribute(new AttributeImpl(IsrAttributes.TARGET_CATEGORY_CODE,
                     getString(node.value)));
             break;
         case NsiliConstants.TARGET_NUMBER:
-            metacard.setAttribute(new AttributeImpl(NsiliMetacardType.TARGET_NUMBER,
+            metacard.setAttribute(new AttributeImpl(IsrAttributes.TARGET_ID,
                     getString(node.value)));
             break;
         case NsiliConstants.TYPE:
-            metacard.setAttribute(new AttributeImpl(NsiliMetacardType.PRODUCT_TYPE,
-                    convertProductType(node.value)));
+            metacard.setAttribute(new AttributeImpl(CoreAttributes.DATATYPE,
+                    getString(node.value)));
```

Comment:
For any of these multi-valued attributes (like datatype) these should be set as a list not as a String value.